### ISSUE TITLE
Multi-line messages

### ISSUE THREAD

**Issue by LukeLR** [vysheng/tg#746]
Is there any way to send multi-line messages now, besides `send_text`?

**Comment by beMoD** [comment]
Yes. Use msg contact "test\ntest2". The key is to use the " around your text and then \n gets interpreted as newline.

**Comment by LukeLR** [comment]
Okay, thanks, I'll give it a try.
